src/osx/iphone/settings.mm

```diff
@@ -18,9 +18,55 @@
 #endif
 
 #include "wx/osx/core/private.h"
+#include "wx/osx/private/available.h"
 
 #include "UIKit/UIKit.h"
 
+#import <Foundation/Foundation.h>
+
+
+static int wxOSXGetUserDefault(NSString* key, int defaultValue)
+{
+    NSUserDefaults* defaults = [NSUserDefaults standardUserDefaults];
+    if (!defaults)
+    {
+        return defaultValue;
+    }
+
+    id setting = [defaults objectForKey: key];
+    if (!setting)
+    {
+        return defaultValue;
+    }
+
+    return [setting intValue];
+}
+
+// ----------------------------------------------------------------------------
+// wxSystemAppearance
+// ----------------------------------------------------------------------------
+
+wxString wxSystemAppearance::GetName() const
+{
+    UITraitCollection* traitCollection = [UIScreen mainScreen].traitCollection;
+    if (traitCollection.userInterfaceStyle == UIUserInterfaceStyleDark)
+        return wxT("Dark");
```

Comment:
We shouldn't use `wxT()` in new code, it should be either `wxS()` or nothing at all.

```suggestion
        return wxS("Dark");
```